freqtrade/rpc/telegram.py

```diff
@@ -196,6 +197,8 @@ def _init(self) -> None:
                                  pattern='update_sell_reason_performance'),
             CallbackQueryHandler(self._mix_tag_performance, pattern='update_mix_tag_performance'),
             CallbackQueryHandler(self._count, pattern='update_count'),
+            CallbackQueryHandler(self._forcebuy_inline, pattern=r"\S+\/\S+"),
+            CallbackQueryHandler(self._forcesell_inline, pattern=r"[0-9]+\s\S+\/\S+"),
```

Comment:
I'll have to ask you to align the naming of the new segments to the new pattern - which will be `forceenter` and `forceexit` instead of *buy and *sell.

Usually, `buy` should map 1:1 to `enter` (same with sell -> exit) - as we used to only do long trades - so an entry was always a buy.

Reply:
No problems, I just used the naming which was already there, but will rename it.

Reply:
O I see you renamed a lot, just as I grasped to understand the code. What about the RPC calls?
Is this done on purpose, doesn't look very consistent naming?

``` 
    def _rpc_forceexit(self, trade_id: str, ordertype: Optional[str] = None) -> Dict[str, str]:
..
    def _rpc_force_entry(self, pair: str, price: Optional[float], *,
```

Reply:
it's more about the name than the "format" itself. It's inconsistent - yes - but it's also not really part of a public interface.